backend/ai_service.py

```
@@ -0,0 +1,364 @@
+            return {
+                "text": "I apologize, but I'm having trouble processing your request right now.",
+                "emotion": "neutral",
+                "confidence": 0.5,
+                "language": language
+            }
+    
+    def _detect_language(self, text: str) -> str:
+        """Detect the language of the input text"""
+        try:
+            return detect(text)
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)